frontend/translations/users/en.json

```diff
@@ -0,0 +1,15 @@
+{
+  "userSearch": "User search",
+  "search": "Search",
+  "noResults": "No results",
+  "userEmail": "Email",
+  "userFirstName": "First name",
+  "userLastName": "Last name",
+  "userStudentNumber": "Student number",
+  "completions": "Completions",
+  "rowsPerPage": "Rows per page:",
+  "displayedRows": "({ from, to, count }) => `${from}-${to === -1 ? count : to} of ${count}`",
```

Comment:
I think a cleaner solution would be if we handled the formatting outside the translation, and that outside logic could only translate the necessary connecting words.

Reply:
Yes, obviously. Just did it like this, because TablePagination wants a function: https://material-ui.com/api/table-pagination/
Other way would be to fork this: https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/locale/index.js